recipes/gdb/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "gdb" %}
+{% set version = "7.11" %}
+{% set sha256 = "7a434116cb630d77bb40776e8f5d3937bed11dea56bafebb4d2bc5dd389fe5c1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: gdb-{{ version }}.tar.xz
+  url: http://ftp.gnu.org/gnu/gdb/gdb-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - python
+    - ncurses ==5.9
+    - texinfo
+    - toolchain
+    - xz 5.2.*
+  run:
+    - python
+    - ncurses >=5.9,<6.0
+    - xz >=5.2,<6.0
```

Comment:
I think as a matter of course, though, we are asking that the pins for certain packages be the same as in the link @183amir shared.  When a feedstock is rerendered, these lines will be rewritten with the pins in that line. So even with the ABI compatabilities you point out, it is just going to get wiped away.